local_planner/src/nodes/common.cpp

```diff
@@ -115,22 +115,22 @@ void createPoseMsg(Eigen::Vector3f& out_waypt, Eigen::Quaternionf& out_q,
 }
 
 float getYawFromQuaternion(const Eigen::Quaternionf q) {
-  float siny_cosp = +2.0 * (q.w() * q.z() + q.x() * q.y());
-  float cosy_cosp = +1.0 - 2.0 * (q.y() * q.y() + q.z() * q.z());
+  float siny_cosp = 2.0 * (q.w() * q.z() + q.x() * q.y());
+  float cosy_cosp = 1.0 - 2.0 * (q.y() * q.y() + q.z() * q.z());
   float yaw = atan2(siny_cosp, cosy_cosp);
 
-  return static_cast<float>(yaw);
+  return static_cast<float>(yaw * RAD_TO_DEG);
```

Comment:
Is this cast needed? I guess `RAD_TO_DEG` is a float already, so you can remove it